src/contract_wrappers/tokens/security_token_wrapper/common.ts

```diff
@@ -2443,6 +2445,9 @@ export default abstract class SecurityTokenCommon extends ERC20TokenWrapper {
     );
     assert.isNonZeroETHAddressHex('Wallet', data.wallet);
     assert.isNonZeroETHAddressHex('ReserveWallet', data.treasuryWallet);
+    data.stableTokens.forEach(address => assert.isNonZeroETHAddressHex('stableTokens', address));
+    data.customOracleAddresses.forEach(address => assert.isNonZeroETHAddressHex('customOracleAddresses', address));
+    assert.assert(data.denominatedCurrency !== '', ErrorCode.InvalidData, 'Denominated Currency not provided');
```

Comment:
Same case here. There are some more complex cases we must take care.
Check my comment for `modifyOracles`.

Reply:
At the momento of creation, Is it necessary to check if the address is ZERO if the fundRaiseType is ETH? I mean... there is no definition before creation.

Reply:
At the moment of creation, theres nothing defined yet, but you are sending all the values to be set.
Those values must be checked.
1) You could send an empty array for customOracleAddresses AND an empty string for denominated currency -> Contract will set default oracles and 'USD' as denominated currency
2) If you are not setting ETH as fundRaiseType, you could send '0x0000...' as address for the first oracle.
3) If you are not setting POLY as fundRaiseType, you could send 0x0000...' as address for the second oracle.